### ISSUE TITLE
[Move devex] Move CLI integration with VScode

### ISSUE THREAD

**Issue by awelc** [MystenLabs/sui#1344]
Provide the ability to build and test Move code from inside IDE using (confgurable) CLI of developer's choice (e.g., core Move CLI or sui-move CLI).

**Comment by stefan-mysten** [comment]
@awelc is this something still relevant?

**Comment by awelc** [comment]
> @awelc is this still relevant?

Yes. I will add this to our plugin at some point in a similar way it's done for rust-analyzer.
